### ISSUE TITLE
Consider using recent downloads for rankings

### ISSUE THREAD

**Issue by loic-sharma** [NuGet/NuGetGallery#7406]
Prototype using recent downloads to rank packages. Target the following queries:

* [blazor](https://www.nuget.org/experiments/search-sxs?q=blazor)
* [codeanalysis.csharp](https://www.nuget.org/experiments/search-sxs?q=codeanalysis.csharp) - `•	Microsoft.CodeAnalysis.CSharp.Features` should be in the results
* [CacheManager](https://www.nuget.org/experiments/search-sxs?q=CacheManager) - `CacheManager.*` packages should be first
* [odata](https://www.nuget.org/packages?q=odata) - The old package [`Microsoft.Data.OData`](https://www.nuget.org/packages/Microsoft.Data.OData/) should not be the first result as it is the older variant.

# Investigation

Analysis done using NuGet Trends. Queries that could be improved using recent downloads:

* [Blazor](https://nugettrends.com/packages?months=3&ids=Microsoft.AspNetCore.Blazor.Cli&ids=BlazorStrap&ids=Blazor.Extensions.Storage&ids=Blazor.Extensions.SignalR) - Recent downloads is a better indicator
* [Entity Framework vs Entity Framework Core](https://nugettrends.com/packages?months=6&ids=EntityFramework&ids=Microsoft.EntityFrameworkCore) - Indicates that Entity Framework should be favored

Queries that would be unaffected by using recent downloads:

* [ASP.NET Core direct vs transitive dependencies](https://nugettrends.com/packages?months=3&ids=Microsoft.AspNetCore.Http.Abstractions&ids=Microsoft.AspNetCore.Http&ids=Microsoft.AspNetCore.Http.Features&ids=Microsoft.Extensions.Configuration&ids=Microsoft.Extensions.DependencyInjection&ids=Microsoft.AspNet.Mvc) - Recent downloads gives no difference
* [MySQL](https://nugettrends.com/packages?months=24&ids=MySql.Data&ids=MySqlConnector&ids=Pomelo.EntityFrameworkCore.MySql&ids=MySql.Data.Entity&ids=MySql.Data.EntityFrameworkCore) - Recent downloads gives no difference
* [Old vs new CosmosDB packages](https://nugettrends.com/packages?months=3&ids=Microsoft.Azure.DocumentDB&ids=Microsoft.Azure.Cosmos)

Queries that would be regressed by using recent downloads: N/A

TODO:
* https://www.nuget.org/experiments/search-sxs?q=Twain

**Comment by tonyqus** [comment]
The search-sxs page looks to be closed since 2019. Do we still need this issue?

![Image](https://github.com/user-attachments/assets/ef7ddbb3-10d3-4677-bc45-fbe400b728ee)

**Comment by joelverhagen** [comment]
Yes. The Search side-by-side page was used temporarily while we were rebuilding the search service on top of Azure Search, to compare to the old method. But the root problem described still exists. We use total download count over recent download count, thus favoring incumbents over newcomers.
